mava/systems/config.py

```diff
@@ -136,13 +137,13 @@ def update(self, **kwargs: Any) -> None:
                         self._config[name] = dataclass
                 else:
                     raise Exception(
-                        f"The given component config ({name}) is not part of the current \
```

Comment:
Why was this removed?

Reply:
It was because merge conflict edit I will restore it